.rubocop.yml

```diff
@@ -1,10 +1,9 @@
 inherit_from:
   - .rubocop_todo.yml
 
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
+inherit_gem:
+  theforeman-rubocop:
+    - strictest.yml
```

Comment:
Should we really use `strictest` given the amount of todo it generated?

Reply:
@evgeni not sure why i used that earlier, but now with default we have less cops in todo as compared to what we have earlier.